crates/rpc/rpc/src/eth/error.rs

```diff
@@ -39,7 +39,12 @@ pub enum EthApiError {
     UnknownBlockNumber,
     /// Thrown when querying for `finalized` or `safe` block before the merge transition is
     /// finalized, <https://github.com/ethereum/execution-apis/blob/6d17705a875e52c26826124c2a8a15ed542aeca2/src/schemas/block.yaml#L109>
-    #[error("unknown block")]
+    ///
+    /// op-node uses case sensitive string comparison to parse this error:
+    /// <https://github.com/ethereum-optimism/optimism/blob/0913776869f6cb2c1218497463d7377cf4de16de/op-service/sources/l2_client.go#L105>
+    ///
+    ///  Temporary, until a version of <https://github.com/ethereum-optimism/optimism/pull/10071> is pushed through that doesn't require this to figure out the EL sync status.
```

Comment:
```suggestion
    /// Temporary, until a version of <https://github.com/ethereum-optimism/optimism/pull/10071> is pushed through that doesn't require this to figure out the EL sync status.
```

and maybe convert to a `TODO`?